core/src/scrim/scrim.service.ts

```diff
@@ -206,7 +206,7 @@ export class ScrimService {
         const franchiseProfiles = await Promise.all(scrim.players.map(async p => {
             const member = await this.memberService.getMember({where: {user: {id: p.id} } });
             const gameMode = await this.gameSkillGroupService.getGameSkillGroupById(scrim.skillGroupId, {relations: ["game"] });
-            const franchise = await this.memberService.getFranchiseByMember(member.id, scrim.organizationId, gameMode.game.id);
+            const franchise = await this.memberService.getFranchiseByMember(member.id, scrim.organizationId, gameMode.game.id).catch(() => null);
```

Comment:
This can be swallowed because it is not at all useful to log that the player isn't on a franchise.

Reply:
Does the `getFranchiseByMember` function _need_ to throw an error if one isn't found? Unsure where else this is used